tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -330,18 +346,15 @@ static ArgumentParser argParser() {
                .type(String.class)
                .metavar("TRANSACTIONAL-ID")
                .dest("transactionalId")
-               .setDefault("performance-producer-default-transactional-id")
-               .help("The transactionalId to use if transaction-duration-ms is > 0. Useful when testing the performance of concurrent transactions.");
+               .help("The transactional id to use. This config take precedence over transactional.id via --producer.config or --producer-props");
```

Comment:
Could you please remind users that `--transaction-duration-ms` will have default value if transaction id is defined.

Reply:
Addressed in the latest commit, thanks.